src/desktop/apps/artist2/components/Meta.js

```diff
@@ -1,4 +1,6 @@
 import React, { Fragment } from 'react'
+import { compactObject } from 'desktop/models/mixins/compact_object.coffee'
```

Comment:
Let's try to avoid relying on util functions written in coffeescript if we can. you could use the `filter` function in  `lodash` version of these instead

Reply:
`filter` is for arrays, but managed to make this work with `pick(object, identity)` -- updated the artwork helper as well.